packages/api/src/external/carequality/process-outbound-patient-discovery-resps.ts

```diff
@@ -97,6 +106,12 @@ export async function processOutboundPatientDiscoveryResps({
         requestId,
         pdLinks: cqLinks.length,
         duration: elapsedTimeFromNow(discoveryParams.startedAt),
+        nonErrorCount,
+        errorCount,
+        httpErrorCount,
+        schemaErrorCount,
+        specificErrorCount,
+        gatewayCounts,
```

Comment:
@Orta21 can we send a json obj like gatewayCounts to posthog and do analytics off of it? My objective is to have a graph with gateway specific success rates over time so we can have a little bit of a more granular view of things